generate_schema/provider/trips.json

```diff
@@ -0,0 +1,181 @@
+                "type": "integer",
+                "description": "The length of time, in seconds, that the trip lasted",
+                "default": 0,
+                "examples": [
+                  600
+                ]
+              },
+              "trip_distance": {
+                "$id": "#/properties/data/properties/trips/items/properties/trip_distance",
+                "type": "integer",
+                "description": "The distance, in meters, that the trip covered",
+                "default": 0,
+                "examples": [
+                  1000
+                ]
+              },
+              "route": {
+                "$id": "#/properties/data/properties/trips/items/properties/route",
+                "title": "The Route Schema",
+                "$ref": "#/definitions/FeatureCollectionMDS"
+              },
+              "accuracy": {
+                "$id": "#/properties/data/properties/trips/items/properties/accuracy",
+                "type": "integer",
+                "title": "The approximate level of accuracy, in meters, of Points within route",
+                "default": 0,
+                "examples": [
+                  15
+                ]
+              },
+              "start_time": {
+                "$id": "#/properties/data/properties/trips/items/properties/start_time",
+                "type": "number",
+                "description": "The time the trip began, expressed as a Unix Timestamp",
+                "default": 0.0,
+                "examples": [
+                  1529968782.421409
+                ]
+              },
+              "end_time": {
+                "$id": "#/properties/data/properties/trips/items/properties/end_time",
+                "type": "number",
+                "description": "The time the trip ended, expressed as a Unix Timestamp",
+                "default": 0.0,
+                "examples": [
+                  1531007628.3774529
+                ]
+              },
+              "parking_verification_url": {
+                "$id": "#/properties/data/properties/trips/items/properties/parking_verification_url",
+                "type": "string",
+                "description": "A URL to a photo (or other evidence) of proper vehicle parking",
+                "default": "",
+                "examples": [
+                  "https://data.provider.co/parking_verify/1234.jpg"
+                ],
+                "pattern": "^(https://.*)$"
+              },
+              "standard_cost": {
+                "$id": "#/properties/data/properties/trips/items/properties/standard_cost",
+                "type": "integer",
+                "description": "The cost, in cents, that it would cost to perform the trip in the standard operation of the System",
+                "default": 0,
+                "examples": [
+                  500
+                ]
+              },
+              "actual_cost": {
```

Comment:
I don't think we have international currencies but we will probably need to extend this at some point in the future.

Reply:
Ah, good point. We could add an additional `currency` field and use the [ISO 4217 standard codes](https://en.wikipedia.org/wiki/ISO_4217). Probably don't need to worry about this just yet though, right?